corehq/warehouse/transforms/sql/user_dim.sql

```diff
@@ -0,0 +1,55 @@
+INSERT INTO {{ user_dim }} (
+    domain,
+    user_id,
+    username,
+    user_type,
+    first_name,
+    last_name,
+    email,
+    doc_type,
+    is_active,
+    is_staff,
+    is_superuser,
+    last_login,
+    date_joined,
+    deleted,
+    dim_last_modified,
+    dim_created_on
+)
+SELECT
+    domain,
+    user_id,
+    username,
+    CASE
+        WHEN user_id = 'system' THEN 'system'
+        WHEN user_id = 'demo_user' THEN 'demo'
+        WHEN user_id = 'commtrack-system' THEN 'supply'
+        WHEN doc_type = 'WebUser' THEN 'web'
+        WHEN doc_type = 'CommCareUser' THEN 'mobile'
+        ELSE 'unknown'
```

Comment:
yea these are used in the report filters. it's what's used when you use the filter "CommCare Supply", etc